Tools/GNUMake/Make.defs

```diff
@@ -733,6 +733,7 @@ endif
 ifeq ($(USE_CONDUIT),TRUE)
   $(info Loading $(AMREX_HOME)/Tools/GNUMake/packages/Make.conduit...)
   include        $(AMREX_HOME)/Tools/GNUMake/packages/Make.conduit
+  include        $(AMREX_HOME)/Src/Extern/Conduit/Make.package
```

Comment:
For consistence, maybe we should include this in the main make file like all other `Make.package` files.

Reply:
Sorry, hadn't seen that. Just committed moving it from Make.defs to Make.conduit.

Reply:
Is the updated version ok? Apart from two commits that I seemed to have accidentally picked up, it only changes package specific files Make.conduit and Make.ascent. I can rebase to avoid picking up the other two commits.